e2e/firstTest.e2e.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-env detox/detox, jest/globals */
 
-import { byId, byText } from "./matcher";
+import { byId, byLabel, byText } from "./matcher";
```

Comment:
```suggestion
import { byId, byText } from "./matcher";
```

lint fix - unused var